### ISSUE TITLE
"Response", should be "MSF Response"

### ISSUE THREAD

**Issue by LucieGueuning** [MSFREACH/msf-reach#653]
cf. field definition issue - cf. file shared in Google drive with field definition

**Comment by matthewberryman** [comment]
@LucieGueuning These are not bugs (aside from some of the wording changes, which I will get onto soon) but changes from what's in the sheet, e.g. it still lists medical figures:
<img width="412" alt="screenshot 2018-10-12 08 51 37" src="https://user-images.githubusercontent.com/2288238/46841695-1d30f300-cdfc-11e8-8fd6-21e308f52ec8.png">
Fields on event open doesn't list population 
<img width="290" alt="screenshot 2018-10-12 08 53 11" src="https://user-images.githubusercontent.com/2288238/46841764-7bf66c80-cdfc-11e8-86e9-bde80bd1a198.png">
(again, care is needed not to overload people and the population figures may come from country information which isn't available until event is opened and CIA Factbook / other country info is available). 
etc.
Since, as previously discussed, field changes take a while and can be complicated to implement, we need to wait until we've got feedback from pilots before proceeding further with the (non-wording) changes.

**Comment by matthewberryman** [comment]
@LucieGueuning operator field done (supersedes previous request in 2b) - please review in test environment. Note that operator is set automatically on event creation.

**Comment by LucieGueuning** [comment]
What do you mean operator field done? 

Please follow the shared file in drive for field definition

**Comment by matthewberryman** [comment]
There are now two fields per comment above:
“1) mission contact person, and (2) REACH operator.”

Sent from my iPhone

On 14 Nov 2018, at 05:32, Lucie Gueuning <notifications@github.com<mailto:notifications@github.com>> wrote:


What do you mean operator field done?

Please follow the shared file in drive for field definition

**Comment by LucieGueuning** [comment]
Please follow the file and not comments above. The file shared that I never stop referring to is the right reference one!

**Comment by matthewberryman** [comment]
> Please follow the file and not comments above. The file shared that I never stop referring to is the right reference one!

Ok, can I suggest you keep consistency with the spreadsheet in future to minimise any errors. 

I have updated the wording to be "REACH Operator" per the spreadsheet but can you please double check that as I thought we were using "MSF REACH" not "REACH" consistently per other discussions?

**Comment by LucieGueuning** [comment]
You are right for consistency but I won't changing the wording again (for you to continue working on other deliverables). REACH operator is acceptable. For that specific operator field one: that's all good and working well. only order that is not followed but I guess it's coming with the field of info cleaned up.

Please follow carefully all the wordings and order for each field of information into the spreadsheet shared. Awaiting for the rest to be cleaned up as per shared spreadsheet.
